packages/arb-validator-core/ethbridge/arbRollupWatcher.go

```diff
@@ -309,28 +406,34 @@ func (vm *ethRollupWatcher) processEvents(chainInfo arbbridge.ChainInfo, ethLog
 			Staker:    common.NewAddressFromEth(eventVal.Staker),
 			NodeHash:  eventVal.NodeHash,
 		}, nil
-	} else if ethLog.Topics[0] == rollupChallengeStartedID {
+	} else if ethLog.Topics[0] == challengeStartedID {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)